client/src/pages/Compliance.tsx

```diff
@@ -0,0 +1,250 @@
+      padding: "7px",
+      textAlign: "center",
+      fontWeight: "bold",
+    },
+    filterContainer: {
+      marginBottom: "10px",
+    },
+    filterElement: {
+      [theme.breakpoints.up("md")]: {
+        margin: "7px",
+        width: "155px",
+      },
+      [theme.breakpoints.down("xl")]: {
+        margin: "7px",
+        width: "120px",
+      },
+      [theme.breakpoints.down("lg")]: {
+        margin: "7px",
+        width: "120px",
+      },
+    },
+    statusFilter: {
+      [theme.breakpoints.up("md")]: {
+        margin: "7px",
+        width: "165px",
+      },
+      [theme.breakpoints.down("xl")]: {
+        margin: "7px",
+        width: "100px",
+      },
+    },
+    filterBtn: {
+      [theme.breakpoints.up("md")]: {
+        marginTop: "12px",
+        marginBottom: "7px",
+        marginRight: "5px",
+        marginLeft: "50px",
+      },
+      [theme.breakpoints.down("xl")]: {
+        marginTop: "15px",
+        marginBottom: "7px",
+        marginRight: "5px",
+        marginLeft: "25px",
+      },
+    },
+    button: {
+      marginTop: "20px",
+      marginBottom: "20px",
+      border: "0",
+      backgroundColor: theme.palette.primary.main,
+      color: "white",
+      "&:hover": {
+        color: "black",
+      },
+    },
+    sortElement: {
+      margin: "7px",
+      width: "205px",
+    },
+    tableContainer: {
+      maxHeight: (window.screen.height - 75) * 0.8,
+    },
+    tableHeadCell: {
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.primary.contrastText,
+    },
+    tableHeadSortBtn: {
+      display: "flex",
+      alignItems: "center",
+    },
+    statusFilterMain: {
+      display: "block",
+      "@media screen and (max-width: 350px)": {
+        display: "none",
+      },
+    },
+    statusFilterAdditional: {
+      display: "none",
+      "@media screen and (max-width: 350px)": {
+        display: "block",
+      },
+    },
+    accountFilterMain: {
+      "@media screen and (orientation:landscape)": {
+        display: "block",
+      },
+      "@media screen and (orientation:portrait)": {
+        display: "none",
+      },
+    },
+    accountFilterAdditional: {
+      "@media screen and (orientation:landscape)": {
+        display: "none",
+      },
+      "@media screen and (orientation:portrait)": {
+        display: "block",
+      },
+    },
+  };
+
+  const [complianceMessage, setComplianceMessage] = useState<string>("");
+  const { auth } = useContext(AuthContext);
+  const { memberDetails } = useMemberDetails(auth.userID!);
+  const [complianceMail, setComplianceMail] = useState<string>("");
+  const [checkedAnonymous, setCheckedAnonymous] = useState(false);
+  const [openDialog, setOpenDialog] = useState(false);
+  const [error, setError] = useState<string>("");
+
+  const handleDialogClose = () => {
+    setOpenDialog(false);
+    setError("");
+  };
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
```
const handleCheckboxChange = (event: ChangeEvent<HTMLInputElement>) => {